addon/helpers/log-activity.ts

```diff
@@ -2,20 +2,23 @@ import Helper from '@ember/component/helper';
 import { inject as service } from '@ember/service';
 import { assert } from '@ember/debug';
 import ActivityTracking from '../services/activity-tracking';
+import { run } from '@ember/runloop';
 
 export default class extends Helper {
   @service declare activityTracking: ActivityTracking;
 
   compute(params: any[]) {
     const [action, actionDescription] = params;
-    assert(
-      '[helper][log-activity] An action is required for this helper to work properly.',
-      typeof action === 'function'
-    );
-    assert(
-      '[helper][log-activity] An actionDescription needs to be passed for the activity-log to make sense.',
-      typeof actionDescription === 'string'
-    );
+    run(() => {
+      assert(
+        '[helper][log-activity] An action is required for this helper to work properly.',
+        typeof action === 'function'
+      );
+      assert(
+        '[helper][log-activity] An actionDescription needs to be passed for the activity-log to make sense.',
+        typeof actionDescription === 'string'
+      );
+    });
```

Comment:
Fixes flacky tests

Reply:
LIAR ! 😂 
I don't believe in any fix for the param assertions ^^